src/Content/Content.php

```diff
@@ -237,15 +237,15 @@ public function toArray(): array
 	}
 
 	/**
-	 * Updates the content and returns
-	 * a cloned object
+	 * Removed in 5.0.0. We only keep it to simplify debugging,
+	 * when this method is still in use in the wild.
 	 *
-	 * @deprecated 5.0.0 Use $model->version()->update() instead.
+	 * Use $model->version()->update() instead.
 	 */
 	public function update(
 		array|null $content = null,
 		bool $overwrite = false
 	): static {
-		throw new Exception('`$content->update()` has been deprecated. Please use `$model->version()->update()` instead');
+		throw new Exception('This is no longer functional. Please use `$model->version()->update()` instead');
```

Comment:
If it's not functional anymore anyways, could we just remove it?

Reply:
The problem is our magic caller. When we remove the method, it would silently fail by returning an empty "update" field.